src/manage_sql_tls_certificates.c

```diff
@@ -180,6 +180,45 @@ tls_certificate_select_columns ()
   return columns;
 }
 
+/**
+ * @brief Get extra_where string for a TLS certificate iterator or count.
+ *
+ * @param[in]  filter           Filter string.
+ *
+ * @return     Newly allocated extra_where string.
+ */
+gchar *
+tls_certificate_extra_where (const char *filter)
+{
+  GString *ret;
+  gchar *report_id;
+
+  ret = g_string_new ("");
+
+  report_id = filter_term_value (filter, "report_id");
+
+  g_message ("report_id=\"%s\"", report_id);
```

Comment:
Is this line intentional?  Should be g_debug?

Reply:
This was just some test output I forgot to remove before committing.
I've removed it in 635558d6b5a02a8e99d5cf7edeaf1a6b7525ad22.